root/communication-rmi/src/main/java/at/fhv/itb5c/rmi/client/RMIClient.java

```diff
@@ -0,0 +1,38 @@
+package at.fhv.itb5c.rmi.client;
+
+import java.rmi.RemoteException;
+
+import at.fhv.itb5c.commons.dto.IUserFactory;
+
+public class RMIClient {
+	private static RMIClient _client;
+	
+	private String _host = "localhost";
+	private int _port = 1337;

```

Comment:
move these variables to a [properties](https://docs.oracle.com/javase/tutorial/essential/environment/properties.html) file